### PR TITLE
Describe acra-authmanager

### PR DESCRIPTION
**Pull request by Lagovas**
There is CLI description for `acra-authmanager` with example of auth.file. Created empty file for `acra-webconfig` to refer to this page at this PR and not break hugo rendering. Content of that page will be overwritten by #78.

### CODE REVIEW

File: acra/configuring-maintaining/general-configuration/acra-authmanager.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+title: acra-authmanager
```
Comment (iamnotacake) [inline acra/configuring-maintaining/general-configuration/acra-authmanager.md:2]: In some places (inside  acra/configuring-maintaining/general-configuration/*.md) we use `AcraFoo`, in others — `acra-foo`.
Just a note that we'll have to end up with a single variant.

File: acra/configuring-maintaining/general-configuration/acra-authmanager.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+bookCollapseSection: true
+---
+
+# acra-authmanager
+
+`acra-authmanager` is CLI utility for [acra-webconfig](/acra/configuring-maintaining/general-configuration/acra-webconfig.md) user management.
```
Comment (iamnotacake) [inline acra/configuring-maintaining/general-configuration/acra-authmanager.md:8]: What about using another form? If it makes actual difference of course.
```suggestion
`acra-authmanager` is CLI utility for [acra-webconfig]({{< ref "/acra/configuring-maintaining/general-configuration/acra-webconfig.md" >}}) user management.
```

File: acra/configuring-maintaining/general-configuration/acra-authmanager.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+`acra-authmanager` updates encrypted file where stored rows in format: 
+```
+<user>:<salt>:<hash_function_parameters>:<hash(salt, password, parameters)>
+<user>:<salt>:<hash_function_parameters>:<hash(salt, password, parameters)>
+<user>:<salt>:<hash_function_parameters>:<hash(salt, password, parameters)>
+``` 
+Each row is separate entry related to distinct user. 
+
+Every password hashed using [Argon2 hash function](https://en.wikipedia.org/wiki/Argon2). You can find current parameter
+values used for hash function in [Acra source code](https://github.com/cossacklabs/acra/blob/master/cmd/constants.go#L34).
```
Comment (iamnotacake) [inline acra/configuring-maintaining/general-configuration/acra-authmanager.md:122]: This link may become a bit outdated at some point.
Another possible solution would be using search link like [this one](https://github.com/cossacklabs/acra/search?q=DefaultWebConfigAuthArgon2Length), although it also points to other usages of the constant.

**Review by iamnotacake — APPROVED** [review]
Looks good, no obvious problems

File: acra/configuring-maintaining/general-configuration/acra-authmanager.md (hunk 3 above)
Comment (Lagovas) [reply acra/configuring-maintaining/general-configuration/acra-authmanager.md]: changed to link with specified release number that will not be changed

File: acra/configuring-maintaining/general-configuration/acra-authmanager.md (hunk 2 above)
Comment (Lagovas) [reply acra/configuring-maintaining/general-configuration/acra-authmanager.md]: thanks, I just missed. copied from IDE and forgot to wrap with tag

**Pull request merged**